geps/gep-1897.md

```diff
@@ -120,12 +137,335 @@ was followed by intense discussion and closed in favor of a downsize in scope.
 
 In January 2023 we closed GEP-1282 and began a new discussion on enumerating TLS use cases in
 [Gateway API TLS Use Cases](#references), for the purposes of a clear definition and separation of concerns.
-This GEP is the outcome of the TLS use cases #4 and #5 in
-[Gateway API TLS Use Cases](#references) as mentioned in the Goals section above.
+This GEP is the outcome of the TLS use case #4 in
+[Gateway API TLS Use Cases](#references) as mentioned in the Immediate Goals section above.
 
 ## API
 
-Details deferred until we reach consensus on what we want to do, and why we want to do this.
+To allow the gateway client to know how to connect to the backend pod, when the backend pod has its own
+certificate, we implement a metaresource named `TLSConnectionPolicy`, that is previously mentioned as an example in
+[GEP-713: Metaresources and PolicyAttachment](https://gateway-api.sigs.k8s.io/geps/gep-713/), and as a hypothetical in the
+[Policy Attachment](https://gateway-api.sigs.k8s.io/references/policy-attachment/#direct-policy-attachment)
```

Comment:
This reference is gone since I updated that page to just point back to GEP-713 (so we only have one place to update).

Reply:
Updated.